selftests/unit/test_loader.py

```diff
@@ -50,6 +50,45 @@ def test(self):
     main()
 """
 
+AVOCADO_TEST_TAGS = """#!/usr/bin/env python
+from avocado import Test
+from avocado import main
+
+import time
+
+class DisabledTest(Test):
```

Comment:
Yeah, I think I'm biased. As a side note, I don't see references to `tests/<test>.py` in the unittests,  but then there are quite a few of them in the functional tests...